client/src/App.js

```diff
@@ -0,0 +1,65 @@
+import React, { useEffect } from 'react';
+import {
+  Route, Routes, useLocation, useNavigate,
+} from 'react-router-dom';
+import { useDispatch, useSelector } from 'react-redux';
+import { fetchAllHomes } from './redux/homesSlice';
+import { Layout } from './components/Layout';
+import { Home } from './components/Home';
+import { Reserve } from './components/Reserve';
+import { Reservations } from './components/Reservations';
+import AddHome from './components/AddHome';
+import RemoveHome from './components/RemoveHome';
+import { HomeDetails } from './components/HomeDetails';
+import { Login } from './components/Login';
+import { Register } from './components/Register';
+import { Splash } from './components/Splash';
+
+const App = () => {
```

Comment:
- [ ] OPTIONAL: Any attempt to reload the page will prompt me to login/register again.  I suggest make it so on reload, the user that has been logged in the browser does not have to login again.

That way, your UX will be vastly improved. 👍

Reply:
Another great suggestion. Thanks for pointing this out! Resolved now ✅ ✅